e2e/cypress/integration/integrations/builtin_commands/common_commands_spec.js

```diff
@@ -27,6 +29,16 @@ describe('I18456 Built-in slash commands: common', () => {
 
                 cy.apiAddUserToTeam(team.id, user2.id);
             });
+
+            const prefixes = ['hello', 'hello', 'hello', 'hello', 'hello', 'hello', 'hello', 'hello'];
+            userGroup = [];
+            prefixes.forEach((prefix) => {
+                cy.apiCreateUser({prefix}).then(({user: otherUser}) => {
+                    const newUser = otherUser;
+                    cy.apiAddUserToTeam(team.id, newUser.id);
+                    userGroup.push(newUser);
+                });
+            });
```

Comment:
I can't see where the prefix is used during test step/verification. Can simply be,
```suggestion
            Cypress._.times(8, () => {
                cy.apiCreateUser().then(({user: otherUser}) => {
                    cy.apiAddUserToTeam(team.id, otherUser.id);
                    userGroup.push(otherUser);
                });
            });
```

Reply:
👍 Great! I just looking for a way to create a specific number of users without using a for-loop.